routes/web.php

```diff
@@ -73,6 +73,7 @@
 
     // Laravel thinks the singular of `leaves` is `leaf`
     // so instead of using a resource, just define the routes
+    Route::get('leaves', 'LeaveController@index');
```

Comment:
do we actually need this? I can't see anywhere it's being used?

Reply:
Nah. Added it in case ebf wanted to review shape, but she's got postman so not needed.